manifests/params.pp

```diff
@@ -28,6 +28,8 @@
   $configure_telegraf     =  true
   $master_list            =  [$::settings::certname]
   $puppetdb_list          =  [$::settings::certname]
+  $postgres_host          =  undef
```

Comment:
Would we want this to default to settings::certname as well? The default value shouldn't be used unless the profile is included right?

Reply:
yeah, I think this would work